code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm

```diff
@@ -162,6 +162,8 @@
 	var/mob/living/carbon/human/H = user
 	H.physiology.brute_mod *= 0.5
 	H.physiology.burn_mod *= 0.5
+	ADD_TRAIT(H, TRAIT_RESISTLOWPRESSURE, "space_adaptation")
+	ADD_TRAIT(H, TRAIT_NOBREATH, "space_adaptation")
```

Comment:
the source for this should be something other than "space_adaptation", or else gaining and losing the space adaptation mutation will also cause you to lose these traits